tekton/runs/nb-test-run.yaml

```diff
@@ -15,34 +15,44 @@ spec:
   - name: test-target-name
     value: manual-testinig
   - name: artifacts-gcs
-    value: gs://kubeflow-ci-deployment/gabrielwen-testing-2
+    value: gs://kubeflow-ci_temp/jlewi_mnist_testing/2020-0619
```

Comment:
Should this be used as default or should we change, looks like something used during development.